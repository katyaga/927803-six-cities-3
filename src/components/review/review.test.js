import React from "react";
import renderer from "react-test-renderer";
import Review from "./review.jsx";

const MockComment = {
  id: 10,
  authorAvatar: `img/avatar-angelina.jpg`,
  authorName: `Sara`,
  rating: 4,
  date: new Date(`2019-12-23 12:00:11`),
  text: `text1`,
};

it(`Should Review render correctly`, () => {
  const tree = renderer
    .create(<Review
      comment={MockComment}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
