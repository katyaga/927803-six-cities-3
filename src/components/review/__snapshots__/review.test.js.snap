// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review render correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar-angelina.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Sara
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      text1
    </p>
    <time
      className="reviews__time"
      dateTime="2019-12-23"
    >
      December 23, 2019
    </time>
  </div>
</li>
`;
